roles/ceph-mgr/tasks/main.yml

```diff
@@ -18,24 +18,32 @@
 
 - name: get enabled modules from ceph-mgr
   command: "{{ docker_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} --format json mgr module ls"
-  register: enabled_ceph_mgr_modules
+  register: _ceph_mgr_modules
   delegate_to: "{{ groups[mon_group_name][0] }}"
   when:
     - ceph_release_num[ceph_release] >= ceph_release_num['luminous']
 
-- name: set _ceph_mgr_modules fact
+- name: set _ceph_mgr_modules fact (convert _ceph_mgr_modules.stdout to a dict)
   set_fact:
-    _ceph_mgr_modules: "{{ enabled_ceph_mgr_modules.get('stdout', '{}') | from_json }}"
+    _ceph_mgr_modules: "{{ _ceph_mgr_modules.get('stdout', '{}') | from_json }}"
+  when:
+    - ceph_release_num[ceph_release] >= ceph_release_num['luminous']
```

Comment:
Shouldn't this be `> ceph_release_num['luminous']` instead?

Reply:
it will be consumed the same way on >= luminous